src/main/java/seedu/address/logic/parser/AddCommandParser.java

```diff
@@ -19,7 +19,7 @@ public class AddCommandParser implements Parser<AddCommand<?>> {
     public AddCommand<?> parse(String args) throws ParseException {
 
         String entity = args.trim().split(" ")[0];
-        String addArgs = args.replace(" " + entity, "");
+        String addArgs = args.replace(entity, "");
```

Comment:
Hmm I didn't see the implication of this change yet, but seems like there is an existing PR that modifies entity parser also.

Reply:
apparently because of the space it didn't correctly replace the entity if there were no args before or after entity.